app/build.gradle.kts

```diff
@@ -57,4 +59,6 @@ dependencies {
     implementation(libs.kotlinx.coroutines.android)
     implementation(libs.kotlinx.coroutines.core)
     implementation(libs.androidx.lifecycle.livedata.ktx)
+
+    implementation(libs.flexbox)
```

Comment:
Is there no built-in option in Android Studio? If not, then this is okay

Reply:
Not really, Linear Layout doesn't really offer this much flexibility. The the switch buttons are not aligned vertically between each other because texts on the right and left are not the same length